lib/models/pair.dart

```diff
@@ -0,0 +1,6 @@
+class Pair<T1, T2> {
```

Comment:
Do we need our own pair class? We could probably just use a final list of dynamic elements with size 2

Reply:
After changing the exercise model, I just forgot to delete this Pair class I made.